python/example/single_cell_detailed_recipe.py

```diff
@@ -142,25 +142,19 @@ def global_properties(self, gid):
 # Pass the probe in a list because that it what single_recipe expects.
 recipe = single_recipe(cell, [probe])
 
-# (4) Create an execution context
-context = arbor.context()
-
-# (5) Create a domain decomposition
-domains = arbor.partition_load_balance(recipe, context)
-
-# (6) Create a simulation
-sim = arbor.simulation(recipe, domains, context)
+# (7) Create a simulation
```

Comment:
Here and for other modified examples, the associated `rst` should be updated (don't forget the change in numbering here).

Reply:
Yes, that's in the queue with all the flake8 and black stuff ;)